packages/components/src/components/DiagramFlamegraph.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="diagram-flame">
+    <div ref="d3Container"></div>
+  </div>
+</template>
+
+<script>
+// namespace import to better support commonjs
+import * as d3 from 'd3';
+import * as Flamegraph from 'd3-flame-graph';
+import { digestEventArray } from '../lib/flamegraph';
+
+const { flamegraph: createFlamegraph } = Flamegraph;
+
+export default {
+  name: 'v-diagram-flame',
+
+  props: {
+    theme: {
+      type: String,
+      default: 'dark',
+      validator: (value) => ['dark', 'light'].includes(value),
+    },
```

Comment:
`theme` isn't used - it can be removed

Reply:
fixup 0e541a34b5dffae101cb8da3ec71c90806c2da8c